src/libstore/filetransfer.cc

```diff
@@ -101,6 +101,7 @@ struct curlFileTransfer : public FileTransfer
                     this->result.data.append(data);
               })
         {
+            requestHeaders = curl_slist_append(requestHeaders, "Accept-Encoding: zstd, br, gzip, deflate");
```

Comment:
```suggestion
            requestHeaders = curl_slist_append(requestHeaders, "Accept-Encoding: zstd, br, gzip, deflate, bzip2, xz");
```

I think we support at least these two (though I've no idea whether adding them is useful :man_shrugging: )

Reply:
`xz` might be, but I am not aware of webserver implementations for that.